VocaDbWeb/Controllers/Api/UserApiController.cs

```diff
@@ -814,5 +814,13 @@ public class PostStatusLimitedModel
 		[Route("{id:int}/status-limited")]
 		[ApiExplorerSettings(IgnoreApi = true)]
 		public void PostStatusLimited(int id, [FromBody] PostStatusLimitedModel model) => _queries.SetUserToLimited(id, model.Reason, WebHelper.GetRealHost(Request), model.CreateReport);
+
+		[Route("{id:int}/followedArtists/{artistId:int}")]
+		public bool GetFollowedArtist(int id, int artistId) => _queries.GetFollowedArtist(id, artistId);

```

Comment:
I would actually consider using ArtistForUserForApiContract for this instead of bool return value.

The reasons are: 1) you can add additional properties there such as notification settings if needed and 2) primitive values such as bool sometimes work poorly with REST APIs, wrapping inside a value is often recommended

Reply:
Should we return `null` if a user is not following an artist?

Reply:
Or maybe ArtistForUserForApiContract with null artist?